src/components/DatePicker/index.ios.js

```diff
@@ -19,18 +20,28 @@ const datepickerPropTypes = {
 class Datepicker extends React.Component {
     constructor(props) {
         super(props);
-
+        const value = DateUtils.getDateAsText(props.value) || DateUtils.getDateAsText(props.defaultValue) || CONST.DATE.MOMENT_FORMAT_STRING;
```

Comment:
Same comments as in `index.android.js`

Reply:
Yes, that should be more apt for other scenarios.